runtime/account_test.go

```diff
@@ -1069,84 +1065,6 @@ func TestRuntimePublicKey(t *testing.T) {
 		assert.Contains(t, err.Error(), "value of type `PublicKey` has no member `validate`")
 	})
 
-	t.Run("IsValid", func(t *testing.T) {
```

Comment:
It would be good to keep the test, and instead of asserting the `IsValid` field, we can test constructing an invalid `PublicKey` inside a cadence code, and checking if the program terminates with a panic.

Some more context to avoid any confusion: There are 3 ways a public-key can be constructed:
- Through a cadence code (this test)
- PublicKey sent by the fvm, and re-constructed at the fvm-cadence interface (below test)
- Passed in as an argument to the script/transaction ([argument passing test](https://github.com/onflow/cadence/pull/1273/files#r755736755))